cmd/placement/main.go

```
@@ -7,10 +7,13 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	utilflag "k8s.io/component-base/cli/flag"
 	"k8s.io/component-base/logs"
 
+	ocmfeature "open-cluster-management.io/api/feature"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)